src/analysis/retail/monk/mistweaver/modules/features/RisingMistBreakdown.tsx

```diff
@@ -0,0 +1,282 @@
+        </ul>
+      </>
+    );
+  }
+
+  vivifyTooltip() {
+    return (
+      <>
+        <SpellLink id={SPELLS.VIVIFY} /> healing from extended{' '}
+        <SpellLink id={talents.RENEWING_MIST_TALENT} /> (
+        {formatPercentage(this.risingMist.extraVivHealing / this.risingMist.totalHealing)}% of
+        Total)
+        <ul>
+          <li>
+            {this.risingMist.extraVivCleaves} extra cleave hits via{' '}
+            <SpellLink id={talents.INVIGORATING_MISTS_TALENT} />
+          </li>
+          <li>
+            {formatNumber(this.risingMist.extraVivhealingFromDancingMistRems)} from extended{' '}
+            <SpellLink id={talents.DANCING_MISTS_TALENT} />{' '}
+            <SpellLink id={talents.RENEWING_MIST_TALENT} />
+          </li>
+          <li>
+            {formatNumber(this.risingMist.extraVivHealingFromHardcastRems)} from extended hardcast{' '}
+            <SpellLink id={talents.RENEWING_MIST_TALENT} />
+          </li>
+          <li>
+            {formatNumber(this.risingMist.extraVivHealingFromRapidDiffusionRems)} from extended{' '}
+            <SpellLink id={talents.RAPID_DIFFUSION_TALENT} />{' '}
+            <SpellLink id={talents.RENEWING_MIST_TALENT} />
+          </li>
+        </ul>
+      </>
+    );
+  }
+
+  essenceFontTooltip() {
+    return (
+      <>
+        Additional <SpellLink id={SPELLS.ESSENCE_FONT_BUFF} /> healing from extensions (
+        {formatPercentage(
+          this.risingMist.essenceFontExtensionHealing / this.risingMist.totalHealing,
+        )}
+        % of Total)
+      </>
+    );
+  }
+
+  gustsOfMistsTooltip() {
+    return (
+      <>
+        Additional <SpellLink id={SPELLS.GUSTS_OF_MISTS} /> healing from extended{' '}
+        <SpellLink id={talents.ESSENCE_FONT_TALENT} /> (
+        {formatPercentage(this.risingMist.extraMasteryhealing / this.risingMist.totalHealing)}% of
+        Total)
+        <ul>
+          {this.selectedCombatant.hasTalent(talents.INVOKE_CHI_JI_THE_RED_CRANE_TALENT) && (
+            <li>
+              {formatNumber(this.risingMist.extraChijiGomHealing)}{' '}
+              <SpellLink id={SPELLS.GUST_OF_MISTS_CHIJI} /> healing from extended{' '}
+              <SpellLink id={talents.ESSENCE_FONT_TALENT} />
+            </li>
+          )}
+          <li>
+            {formatNumber(this.risingMist.extraGomHealing)} <SpellLink id={SPELLS.GUSTS_OF_MISTS} />{' '}
+            healing from extended <SpellLink id={talents.ESSENCE_FONT_TALENT} />
+          </li>
+        </ul>
+      </>
+    );
+  }
+
+  statistic() {
+    return (
+      <TalentAggregateStatisticContainer
+        title={
+          <>
+            <SpellLink id={talents.RISING_MIST_TALENT.id} /> Breakdown -{' '}
+            <ItemHealingDone amount={this.risingMist.totalHealing} />
+          </>
+        }
+        category={STATISTIC_CATEGORY.GENERAL}
+        position={STATISTIC_ORDER.CORE(1)}
```

Comment:
Should be TALENTS

Reply:
I think its ok to leave the other stat and keep this one in CORE similar to our multiple misty peaks statistics. I stripped out the duplicate stuff and left the total time extended per hot. Not married to any particular way forward though

Reply:
It just feels kinda redundant since we show the overall RM healing at the top of this graph

Reply:
KK will do, moved the extension time per hot data into the overall tooltip for the new stat